### PR TITLE
RIA-8258 - Mark appeal as remitted Notification to LR/HO

### PR DESCRIPTION
**Pull request by robin-ieong**
### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/RIA-8258

### Change description ###
Mark appeal as remitted - Notification to LR/HO

### Checklist

- [x] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/config/NotificationHandlerConfiguration.java [hunk 1]
```diff
@@ -3997,6 +3997,20 @@ && isAipJourney(asylumCase)
...
 
+    @Bean
+    public PreSubmitCallbackHandler<AsylumCase> markAppealAsRemittedLrAndHoNotificationHandler(
+        @Qualifier("markAppealAsRemittedLrAndHoNotificationGenerator") List<NotificationGenerator> notificationGenerators
+    ) {
+
+        return new NotificationHandler(
+            (callbackStage, callback) -> {
+                return (callbackStage == PreSubmitCallbackStage.ABOUT_TO_SUBMIT
+                    && callback.getEvent() == Event.MARK_APPEAL_AS_REMITTED);
```
Comment (ThomasKKC) [inline src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/config/NotificationHandlerConfiguration.java:4008]: We can probably add not AIP check here, there is another ticket for AIP.

File: src/functionalTest/resources/scenarios/RIA-8258-mark-appeal-as-remitted.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    "uri": "/asylum/ccdAboutToSubmit",
+    "credentials": "AdminOfficer",
+    "input": {
+      "id": 8258,
+      "eventId": "markAppealAsRemitted",
+      "state": "ftpaDecided",
+      "caseData": {
+        "template": "minimal-appeal-submitted.json",
+        "replacements": {
+          "isDlrmSetAsideEnabled": "Yes",
```
Comment (ThomasKKC) [inline src/functionalTest/resources/scenarios/RIA-8258-mark-appeal-as-remitted.json:14]: Don't need isDlrmSetAsideEnabled and updateTribunalDecisionList here

**Review by ThomasKKC — CHANGES_REQUESTED** [review]

File: src/functionalTest/resources/scenarios/RIA-8258-mark-appeal-as-remitted.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+          "listCaseHearingCentre": "taylorHouse",
```
Comment (MiguelMartinRojas) [inline src/functionalTest/resources/scenarios/RIA-8258-mark-appeal-as-remitted.json:15]: Shouldn't we use a feature flag for this new notification

Comment (ThomasKKC) [reply src/functionalTest/resources/scenarios/RIA-8258-mark-appeal-as-remitted.json:15]: Need to set up a new LD flag, we don't have one yet for Remitted

Comment (robin-ieong) [reply src/functionalTest/resources/scenarios/RIA-8258-mark-appeal-as-remitted.json:14]: added new flag check in the FT

**Pull request merged**